src/System.Private.CoreLib/shared/System/Text/ValueStringBuilder.cs

```diff
@@ -296,5 +296,329 @@ public void Dispose()
                 ArrayPool<char>.Shared.Return(toReturn);
             }
         }
+
+
+        public void AppendFormat(string format, object? arg0) => AppendFormatHelper(null, format, new ParamsArray(arg0));
```

Comment:
This should to in a separate partial class file. Otherwise, all places that use ValueStringBuilder in CoreFX would need include a lot of extra stuff for CoreLib.

Reply:
Moved